fullscreen.bs

```diff
@@ -366,8 +359,16 @@ attribute's getter must run these steps:
    <a lt="unfullscreen a document">unfullscreen <var>descendantDoc</var></a>.
    <!-- cross-process -->
 
-   <li><p>For each <var>exitDoc</var> in <var>exitDocs</var>, in order,
-   <a lt="unfullscreen an element">unfullscreen</a> <var>exitDoc</var>'s <a>fullscreen element</a>.
+   <li>
+    <p>For each <var>exitDoc</var> in <var>exitDocs</var>, in order:
```

Comment:
We could make this use https://infra.spec.whatwg.org/#list-iterate and no longer have to say "in order", but could also do that later on.